src/.nuget/dir.targets

```diff
@@ -39,9 +39,16 @@
       </NativeWithSymbolFile>
     </ItemGroup>
 
+    <ItemGroup>
+      <!-- The symbols for these files are already in place together with respective *.ni.pdb -->
+      <IlForCrossGenedFile Include="@(CrossGenBinary -> '%(RootDir)%(Directory)IL\%(Filename).dll')">
+        <TargetPath>runtimes/$(PackageTargetRuntime)/il</TargetPath>
+      </IlForCrossGenedFile>
+    </ItemGroup>
+
     <ItemGroup>
       <File Include="@(NativeWithSymbolFile)" />
-      <File Include="@(LongNameFile)">
+      <File Include="@(LongNameFile);@(IlForCrossGenedFile)">
         <IsSymbolFile>true</IsSymbolFile>
```

Comment:
Should IsSymbolFile be set to trun for this file?

Reply:
While technically not a symbol file the IL dll is being used only in conjunction with its pdb and since there are other dlls being marked as symbols I followed the same pattern.